.golangci.yaml

```diff
@@ -40,7 +31,6 @@ linters-settings:
   gosimple:
     checks: ['all']
   govet:
-    check-shadowing: true
```

Comment:
While this check can be annoying at times (`err` variable tends to get tangled up in it more often than I'd like) I've seen it as a net win for keeping less experienced Go developers on the rails and avoiding some of Go's rougher edges.

Reply:
It does not seem to be a valid option no more (got a warning in goland). I'll look it up at the source, though.

Reply:
Indeed, `golangci-lint` says:

```
WARN [config_reader] The configuration option `linters.govet.check-shadowing` is deprecated. Please enable `shadow` instead, if you are not using `enable-all`. 
```

So it's valid, but could be renamed.

Reply:
We are using `enable-all`, though, so it's redundant.